src/Tiny/Tiny.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled, { css } from "styled-components";
+
+const TYPE_BASIC = "basic";
+const TYPE_BOLD = "bold";
+
+const CaptionStyles = styled.p`
+  font-size: 12px;
+
+  ${({ type }) =>
+    type === TYPE_BOLD &&
+    css`
+      font-weight: 500;
+    `}
+`;
+
+export default function Caption({ children, style, type }) {
+  return (
+    <CaptionStyles style={style} type={type}>
+      {children}
+    </CaptionStyles>
+  );
+}
+
+Caption.propTypes = {
+  children: PropTypes.string.isRequired,
+  type: PropTypes.oneOf([TYPE_BASIC, TYPE_BOLD]),
```

Comment:
any reason for`bold` instead of `strong` as in the design library?

Reply:
Also should we add monospaced paragraph type in this PR too?

Reply:
> Also should we add monospaced paragraph type in this PR too?

 talked with piotr he said we dont need it

Reply:
> any reason forbold instead of strong as in the design library?

try to keep the api same with the css api. you dont say strong when you style and I am worried that its gonna be something more we need to memorise. Thats the logic. we could change it ofc

Reply:
>try to keep the api same with the css api. you dont say strong when you style and I am worried that its gonna be something more we need to memorise. Thats the logic. we could change it ofc

Sounds good 👍